GameServer/Logic/API/ApiFunctionManager.cs

```diff
@@ -103,18 +103,32 @@ public static void SetChampionModel(Champion champion, string model)
             {
                 var newCoords = _game.Map.AIMesh.getClosestTerrainExit(new Vector2(target.X, target.Y));
                 var newTarget = new Target(newCoords);
-                unit.DashTo(newTarget, dashSpeed, followTargetMaxDistance, backDistance, travelTime);
-                _game.PacketNotifier.notifyDash(unit, newTarget, dashSpeed, keepFacingLastDirection, leapHeight, followTargetMaxDistance, backDistance, travelTime);
+                unit.DashToTarget(newTarget, dashSpeed, followTargetMaxDistance, backDistance, travelTime);
+                _game.PacketNotifier.notifyDash(unit,
```

Comment:
I'd prefer these to be formatted like so:
```CS
_game.PacketNotifier.notifyDash(
    unit,
    newTarget,
    dashSpeed,
    keepFacingLastDirection,
    leapHeight,
    followTargetMaxDistance,
    backDistance,
    travelTime
);
```
Open for discussion, although I'm pretty sure this format is more common in the rest of the code